### PR TITLE
Fix/14954 Prevent presenting popup in EOL

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Prevent presenting  Exposure notification permission popup in EOL

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14954

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Models/Exposure/__tests__/ExposureManagerTests.swift [hunk 1]
```diff
@@ -7,6 +7,21 @@ import ExposureNotification
 @testable import ENA
 
 class ExposureManagerTests: CWATestCase {
+	
+	func test_requesting_userPermission_in_EOL() {
+		UserDefaults.standard.setValue(true, forKey: CWAHibernationProvider.isHibernationInUnitTest)
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Models/Exposure/__tests__/ExposureManagerTests.swift:13]:
```suggestion
                // GIVEN
		UserDefaults.standard.setValue(true, forKey: CWAHibernationProvider.isHibernationInUnitTest)
```

File: src/xcode/ENA/ENA/Source/Models/Exposure/__tests__/ExposureManagerTests.swift [hunk 2]
```diff
@@ -7,6 +7,21 @@ import ExposureNotification
...
+		let managerSpy = ManagerSpy()
+		let exposureManager = ENAExposureManager(manager: managerSpy)
+		let expectation = expectation(description: "Risk permission in EOl should return nil")
+	
+		exposureManager.enable { error in
+			UserDefaults.standard.setValue(false, forKey: CWAHibernationProvider.isHibernationInUnitTest)
+			XCTAssertNil(error)
+			expectation.fulfill()
+		}
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Models/Exposure/__tests__/ExposureManagerTests.swift]:
```suggestion
                // WHEN
		exposureManager.enable { error in
			UserDefaults.standard.setValue(false, forKey: CWAHibernationProvider.isHibernationInUnitTest)
			
			// THEN
			XCTAssertNil(error)
			expectation.fulfill()
		}
```

File: src/xcode/ENA/ENA/Source/Models/Exposure/__tests__/ExposureManagerTests.swift [hunk 3]
```diff
@@ -7,6 +7,21 @@ import ExposureNotification
 @testable import ENA
 
 class ExposureManagerTests: CWATestCase {
+	
+	func test_requesting_userPermission_in_EOL() {
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Models/Exposure/__tests__/ExposureManagerTests.swift]:
```suggestion
	func testRequestEnableUserPermission_EOL_noError() {
```

or something in this direction ...

**Review by flxschmidt — APPROVED** [review]
LGTM. Some suggestions

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**